game.py

```diff
@@ -40,7 +46,7 @@ def print_bar(title, curr, max, level, color='white'):
     progress_earned = int(curr * scale) * '\u2588'
     progress_remaining = int((max - curr) * scale) * '\u2591'
     print(title + level_text)
-    cprint('XP: ' + progress_earned + progress_remaining + ' ' + str(curr) + '/' + str(max) + '\n', color)
+    cprint(progress_earned + progress_remaining + ' ' + str(curr) + '/' + str(max) + '\n', color)
```

Comment:
'XP:' was being applied to the Health and Energy bar. This was the easiest way to fix it. Okay for now 🤷